staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/admission.go

```diff
@@ -0,0 +1,132 @@
+package cel
+
+import (
+	"context"
+	"errors"
+	"io"
+
+	"k8s.io/apiserver/pkg/admission"
+	genericadmissioninitailizer "k8s.io/apiserver/pkg/admission/initializer"
+	genericfeatures "k8s.io/apiserver/pkg/features"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/component-base/featuregate"
+)
+
+////////////////////////////////////////////////////////////////////////////////
+// Plugin Definition
+////////////////////////////////////////////////////////////////////////////////
+
+// Definition for CEL admission plugin. This is the entry point into the
+// CEL admission control system.
+//
+// Each plugin is asked to validate every object update.
+
+const (
+	// PluginName indicates the name of admission plug-in
+	PluginName = "AlphaCEL"
```

Comment:
I think we'll want to initially name the plugin based on what we think the name will be in GA. Prefixing `Alpha` will require renaming the plugin later and breaking existing users of the plugin. The feature gate should be sufficient to let users know that they are opting into an Alpha admisison plugin.